File: 3168/Solution.java
class Solution {
    public int minimumChairs(String s) {
        int res = 0;
        int max = 0;
        for(char c : s.toCharArray()){
            res += c == 'E' ? 1 : -1;
            max = Math.max(max, res);
        }
        return max;
    }
}

File: 3168/Solution.go
func minimumChairs(s string) int {
    count, max := 0, 0
    for _, c := range s{
        if c == 'E'{
            count++
        }else{
            count--
        }
        if count > max {
            max = count
        }
    }
    return max
}
